### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by Honey14**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
 - Fix `ContactPatientBottomSheet` not going back to call patient view on back click in call later mode
 - Fix overdue list not changing when switching facility from overdue screen
   
-## On Demo
+## 2021-07-13-7862
```
Comment (msasikanth) [inline CHANGELOG.md:37]: It looks like we haven't moved the hotfixed changelog into the demo section. Can we do that as well, please?

File: CHANGELOG.md [hunk 2]
```diff
@@ -69,6 +68,9 @@
 - Tap outside or swipe to dismiss the bottom sheets
 - Tap outside to dismiss the dialogs
 
+### Fixes
```
Comment (msasikanth) [inline CHANGELOG.md:71]: We have hot fixed 4 more commits to that release. Let's add those as well (please refer the commits from the release/2021-07-12 branch)

```
- Clean up `gradle.properties`
- Extract product flavor build config fields into `defaultConfig`
- Push records to server during sync in batches
- Allow push and pull sync batch sizes to be configured separately
```

**Review by msasikanth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (vinaysshenoy) [reply CHANGELOG.md:71]: The sync records need to go into a new release since they were hotfixed _after_ the current release was rolled to 25%.

Comment (vinaysshenoy) [reply CHANGELOG.md:71]: I'll make that change once the changes are live.

**Review by msasikanth — APPROVED** [review]

**Pull request merged**